app/models/katello/subscription.rb

```diff
@@ -53,5 +55,11 @@ def virt_who?
     def self.humanize_class_name(_name = nil)
       _("Subscription")
     end
+
+    def notification_recipients_ids
+      User.unscoped.all.find_all do |user|
+        user.can?(:import_manifest, organization_id)
```

Comment:
i got an error with this, it seemed that an organization object needed to be passed in, not its id.